src/components/gas/GasSpeedButton.js

```diff
@@ -254,20 +257,18 @@ const GasSpeedButton = ({ onCustomGasBlur, onCustomGasFocus, type }) => {
         buttons: [
           {
             onPress: complete,
-            text: 'Yes',
+            text: 'Proceed Anyway',
           },
           {
             onPress: () => inputRef.current?.focus(),
             style: 'cancel',
-            text: 'No, thanks',
+            text: 'Edit Gas Price',
           },
         ],
-        message:
-          (tooLow
-            ? 'The gas price you set is too low and your transaction might get stuck or dropped.'
-            : 'We noticed the gas price you set is quite high.') +
-          '\n\nDo you want to continue anyway?',
-        title: 'Heads up!',
+        message: tooLow
+          ? 'If you don’t set a high enough gas price, your transaction '
```

Comment:
This phrase is incomplete

Reply:
forgot to finish these error messages, good catch

will fix